cpp/src/Slice/Parser.cpp

```diff
@@ -1562,28 +1525,20 @@ Slice::Container::createDictionary(const string& name, const TypePtr& keyType, c
     ContainedList matches = _unit->findContents(thisScope() + name);
     if(!matches.empty())
     {
-        DictionaryPtr p = dynamic_pointer_cast<Dictionary>(matches.front());
-        if(p)
-        {
-            if(_unit->ignRedefs())
```

Comment:
ignRedefs was always false, I think it was used before by local Slice and scripting languages.